test/specs/rules/require-operator-before-line-break.js

```diff
@@ -65,4 +66,21 @@ describe('rules/require-operator-before-line-break', function() {
         checker.configure({ requireOperatorBeforeLineBreak: true });
         assert(checker.checkString('var err = "Cannot call " + modelName \n + " !";').getErrorCount() === 1);
     });
+
+    reportAndFix({
+        name: 'should fix operator by moving to previous line',
+        rules: { requireOperatorBeforeLineBreak: true },
+        input: [
+          'if (x',
+          '    && y) {',
+          '  alert(z);',
+          '}'
+        ].join('\n'),
+        output: [
+          'if (x &&',
+          '    y) {',
```

Comment:
Please add cases with `/* comment before ampersands */ &&` and `x // line comment between var and ampersand`